bin/run-nextclade

```diff
@@ -40,6 +46,8 @@ echo "[ INFO] ${0}:${LINENO}: Running Nextclade"
   --input-fasta="${INPUT_FASTA}" \
   --input-dataset="${NEXTCLADE_DATASET_DIR}" \
   --output-tsv="${OUTPUT_TSV}" \
-  --genes=E,M,N,ORF1a,ORF1b,ORF3a,ORF6,ORF7a,ORF7b,ORF8,ORF9b,S \
+  --genes="${GENES}" \
   --output-dir="${NEXTCLADE_OUTPUTS_DIR}" \
-  --output-basename="nextclade"
+  --output-basename="nextclade" \
+  --output-fasta="${OUTPUT_FASTA}" \
+  --output-insertions="${OUTPUT_INSERTIONS}" \
```

Comment:
`run-nextclade` script have gained 3 parameters:
 - OUTPUT_FASTA - this is where the newly aligned sequences end up
 - OUTPUT_INSERTIONS - same for insertions (needed for `mutation-summary` script)
 - GENES - list of genes. Nexclade is using comma-delimited list, while `mutation-summary` script prefers spacede-limited for some reason. We might make `mutation-summary` to reconsider its poor and random behavior :)